packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -311,15 +359,15 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
       ..color = trackColor.withOpacity(trackColor.opacity * fadeoutOpacityAnimation.value);
   }
 
-  void _paintScrollbar(Canvas canvas, Size size, double thumbExtent, AxisDirection direction) {
+  void _paintScrollbar(Canvas canvas, Size size, double thumbExtent, AxisDirection direction, ScrollbarOrientation? scrollbarOrientation){
```

Comment:
```suggestion
  void _paintScrollbar(Canvas canvas, Size size, double thumbExtent, AxisDirection direction){
```

Since scrollbarOrientation is a class member you don't have to pass it.